models/issue_label.go

```diff
@@ -336,6 +432,135 @@ func GetLabelsByRepoID(repoID int64, sortType string, listOptions ListOptions) (
 	return getLabelsByRepoID(x, repoID, sortType, listOptions)
 }
 
+// ________
+// \_____  \_______  ____
+//  /   |   \_  __ \/ ___\
+// /    |    \  | \/ /_/  >
+// \_______  /__|  \___  /
+//         \/     /_____/
+
+// getLabelInOrgByName returns a label by Name in given organization
+func getLabelInOrgByName(e Engine, orgID int64, labelName string) (*Label, error) {
+	if len(labelName) == 0 || orgID <= 0 {
+		return nil, ErrOrgLabelNotExist{0, orgID}
+	}
+
+	l := &Label{
+		Name:   labelName,
+		RepoID: 0,
```

Comment:
Once you made sure orgID is positive, setting `RepoID` to `0` is not required, as `0` is the default value for `int64` anyway.

Reply:
fixed